cme/lms/static/sass/_overrides.scss

```diff
@@ -89,19 +89,22 @@
                 background-image: -webkit-linear-gradient(#D1D5D3 0%, #D1D5D3);
             }
 
-            a img {
-                margin-top: -6px;
+            a {
+                line-height: 1em;
+
+                img {
+                    margin-top: -6px;
+                }
             }
         }
         h2.course-header {
+            margin: 9px 0 0 0;
             float: left;
             font-family: $sans-serif;
             font-size: 0.9em;
             font-weight: 600;
             color: #515553;
             letter-spacing: 0;
-            margin: 0;
-            margin-top: 9px;
```

Comment:
Why this change now?

Reply:
Cause the way we did it before was just stupid?
Also I moved it to a more natural position for `margin`

Reply:
Ahhhhh, I got it now.